WooCommerce/Classes/ViewRelated/Toolbar/ToolbarView.swift

```diff
@@ -18,8 +18,8 @@ final class ToolbarView: UIView {
         pinSubviewToAllEdges(stackView)
     }
 
-    required init(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    required convenience init(coder: NSCoder) {
+        self.init()
```

Comment:
nit: do you think it's safer to call the same `super. init(coder: coder)` and share the other init code in `init()` by another function? (like `commonInit`)

Reply:
Yes, it's safer and makes more sense :) Done in https://github.com/woocommerce/woocommerce-ios/pull/6698/commits/da132fac82ff99a97c8221865e9eb90beaefda8d